tests/Tests/100__deploy/100__installation/109__operator.robot

```diff
@@ -0,0 +1,53 @@
+*** Settings ***
+Documentation       Post install test cases that verify OCP operator resources and objects
+
+Library             OpenShiftLibrary
+Resource            ../../../Resources/ODS.robot
+
+*** Test Cases ***
+Verify Odh-deployer Checks Cluster Platform Type
+    [Documentation]    Verifies if odh-deployer checks the platform type of the cluster before installing
+    [Tags]    Sanity
+    ...       Tier1
+    ...       ODS-1316
+    ${cluster_platform_type}=    Fetch Cluster Platform Type
+    IF    "${cluster_platform_type}" == "OpenStack"
+        ${odhdeployer_logs_content}=    Set Variable    INFO: Deploying on PSI. Creating local database
+    ELSE IF    "${cluster_platform_type}" == "AWS"
+        ${odhdeployer_logs_content}=     Set Variable    INFO: Deploying on AWS. Creating CRO for deployment of RDS Instance
+    ELSE
+        ${odhdeployer_logs_content}=     Set Variable    ERROR: Deploying on ${cluster_platform_type}, which is not supported. Failing Installation
```

Comment:
I think line 17 and 19 are too long as per robocop. Cold you changes to new line with the help of **...** line

Reply:
Done